chart/templates/ingress.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
This takes care of the rancher's ingress. But there are a lot other ingress usages withing rancher code - how are we transitioning them?

Reply:
since we aren't wanting to make significant changes to the code base for 2.5.2, we are just addressing the rancher chart with this issue, and then I have opened this issue https://github.com/rancher/rancher/issues/29482 for the rancher code

Reply:
There is already an issue for this (that I can't find)

Reply:
Ok, lgtm